### ISSUE TITLE
OpenEVSE App

### ISSUE THREAD

**Issue by glynhudson** [emoncms/app#41]
After discussion with @chris1howell it would be great if the OpenEVSE app could display the following:

- Energy per session
- Cost per session
- Temperature graph
- Realtime state indicator (e.g. charging / EV connected / sleeping etc)
- Realtime current

OpenEVSE Wifi currently posts to following to Emoncms 

![emoncms - input view](https://user-images.githubusercontent.com/758844/31748988-f8f8a296-b46e-11e7-84c4-faee7b3fcef9.png)

**Comment by teslatradeup** [comment]
This would be awesome!! Was just thinking about this... I'd love an Android app that displays exactly the same information the LED display shows on the OpenEVSE... Perhaps as a banner at the top of my device. :)

**Comment by glynhudson** [comment]
Are you aware the OpenEVSE already has a web interface? Here is a demo: https://openevse.openenergymonitor.org/

**Comment by teslatradeup** [comment]
Yes I do. But it would be awesome to have an app that puts your current
state in the notification area on a mobile device with notification options.

Same info as what's on the OpenEVSE itself. :)

Is that information available somehow with credentials locally through an
API?

**Comment by teslatradeup** [comment]
Would be awesome if we could see the info straight up like this:
![OpenEVSE Mockup](https://user-images.githubusercontent.com/43191851/57995491-5a471d00-7a90-11e9-8c29-d50f807b9265.png)

**Comment by glynhudson** [comment]
Agree, that would be *awesome*. We would need a native app for that. If you know anyone with Android development experience who's interested in the OpenEVSE project please put them in contact.
